examples/convert_to_aster.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2018 COAL Developers
+#
+# This program is free software; you can redistribute it and/or 
+# modify it under the terms of the GNU General Public License 
+# as published by the Free Software Foundation; version 2.
+#
+# This program is distributed in the hope that it will be useful, 
+# but WITHOUT ANY WARRANTY; without even the implied warranty 
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public 
+# License along with this program; if not, write to the Free 
+# Software Foundation, Inc., 51 Franklin Street, Fifth 
+# Floor, Boston, MA 02110-1301, USA.
+# encoding: utf-8
+
+'''
+Spectral Version 7 .txt file converted to ASTER .txt file format
```

Comment:
```Spectral Version 7``` doesn't refer to anything specific... make it specific.

Reply:
Additionally, **THERE IS NO ASTER .txt FILE FORMAT**... the input is a DB file. I've already explained this several times.